### PR TITLE
Fix wallet connect login

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2345

### CODE REVIEW

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 1]
```diff
@@ -170,6 +170,7 @@ export const EVMHomeAdaptorProvider = ({
     const selectedWallet = localStorage.getItem(ONBOARD_SELECTED_WALLET) as string;
     let connected = false;
 
+    console.log('wallet select init');
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Do we need this log?

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx [hunk 2]
```diff
@@ -187,7 +188,11 @@ export const EVMHomeAdaptorProvider = ({
             connected = false;
           })
           .finally(() => {
-            if (!connected) setWalletType("unset");
+            if (!connected) {
+              resetOnboard();
+              setWalletType("unset");
+              console.log('finished wallet connection')
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: Please, update log message

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 1 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: It can be useful for debugging

Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: removed

File: src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx]: updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**